.travis.yml

```diff
@@ -5,10 +5,23 @@ python:
   - "3.4"
   - "3.5"
   - "3.6"
-install:
-- travis_retry pip install tox-travis
+  - "pypy"
```

Comment:
I would like to see `pypy` added in appveyor (for windows) too, in order to keep the tests the same.